aerospike/datadog_checks/aerospike/check.py

```diff
@@ -0,0 +1,32 @@
+# (C) Datadog, Inc. 2022-present
+# (C) 2018 Aerospike, Inc.
+# (C) 2017 Red Sift
+# (C) 2015 Pippio, Inc. All rights reserved.
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+from datadog_checks.base import OpenMetricsBaseCheckV2
+
+from .metrics import METRIC_MAP
+
+
+class AerospikeCheckV2(OpenMetricsBaseCheckV2):
+
+    __NAMESPACE__ = 'aerospike'
+
+    DEFAULT_METRIC_LIMIT = 0
+
+    def __init__(self, name, init_config, instances):
+        super().__init__(name, init_config, instances)
+
+        if 'metrics' in self.instance:
```

Comment:
There should be a `validators.py` file in the `config_models/` directory where this validation can occur. Can you rerun `ddev validate aerospike -s` to check if it generates the `validators.py` file? You may need to rebase from `master`.

Reply:
Ah okay, I see now that makes sense. I just moved it over to the validators.py- thanks for the tip!

Reply:
Moved and refactored the test to check that a ValueError is thrown